subsys/jaxrs/src/main/java/org/commonjava/indy/bind/jaxrs/ApiVersioningFilter.java

```diff
@@ -104,7 +104,7 @@ public void doFilter( final ServletRequest request, final ServletResponse respon
         String reqApiVersion = httpServletRequest.getHeader( HEADER_INDY_API_VERSION );
 
         // Insert 3 headers into outgoing responses
-        httpServletResponse.addHeader( HEADER_INDY_API_VERSION, reqApiVersion );
+        // httpServletResponse.addHeader( HEADER_INDY_API_VERSION, reqApiVersion );
```

Comment:
So why is this not needed?

Reply:
This is the entry from request, I checked and looks it's not used. If it's the case, no need to add the extra remediation(whitelist as recommended) to check if the value is in the whitelist.

Reply:
Ok, so it sounds good now.